internal/internal_workflow_testsuite.go

```diff
@@ -818,6 +818,7 @@ func (env *testWorkflowEnvironmentImpl) Complete(result []byte, err error) {
 		env.logger.Debug("Workflow already completed.")
 		return
 	}
+	env.workflowDef.Close()
```

Comment:
do we have to close it in the very beginning of here?

Reply:
meaning prior to line 817, or wondering if it can be / would be better to do later down in this func?

for earlier: shouldn't be necessary... I think.  I'm not entirely sure how these are called / what about crons?

for later: I didn't test other locations, but I think it failed if it was at the _end_.  I can try others if you think it may be troublesome to do here.